app.js

```diff
@@ -0,0 +1,118 @@
+/** @format */
+
+'use strict';
+{
+  // 要素を取得
+  const btnsDiv = document.getElementById('buttons');
+  const btnStart = document.querySelector('.btnStart');
+  const head = document.querySelector('.header');
+  const question = document.querySelector('.question');
+  const genre = document.querySelector('.genre');
+  const difficult = document.querySelector('.difficult');
+
+  // 変数を定義
+  let val = 0;
+  let correctVal = 0;
+
+  // 開始ボタンのイベント
+  btnStart.addEventListener('click', () => {
+    getQuiz(val);
+  });
```

Comment:
非同期処理を呼び出す場合は `await` を使う必要があり
`await` を利用する場合は呼び出しを行う関数に `async` が必要です。

```suggestion
  btnStart.addEventListener('click', async () => {
    await getQuiz(val);
  });
```